.circleci/wait-for-job-finish.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# Exit script if you try to use an uninitialized variable.
+set -o nounset
+
+# Use the error status of the first failure, rather than that of the last item in a pipeline.
+set -o pipefail
+
+x=0
```

Comment:
This variable could use a more descriptive name.

Reply:
Changed to `counter`.